ajax/libs/jQuery.Gantt/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "jQuery.Gantt",
+  "version": "1.2.2",
+  "description": "jQuery Gantt Chart is a simple chart that implements gantt functionality as a jQuery component.",
```

Comment:
I think both are fine. https://github.com/mbielanczuk/jQuery.Gantt#demo-and-documentation